src/main/scala/mesosphere/marathon/core/task/tracker/impl/TaskLoader.scala

```diff
@@ -0,0 +1,9 @@
+package mesosphere.marathon.core.task.tracker.impl
+import scala.concurrent.Future
+
+/**
+  * Loads all task data into an [[AppDataMap]].
+  */
+private[tracker] trait TaskLoader {
```

Comment:
Generally I like traits very much. Do we want to ouse them also internal object that are not used out of this context?